### PR TITLE
Promote images for alpha versions of kops

### PR DESCRIPTION
**Pull request by justinsb**
* GCE
* kops 1.11 (not yet released)

### CODE REVIEW

File: channels/stable [hunk 1]
```diff
@@ -23,9 +23,12 @@ spec:
     # BUT... this is causing the submit queue to block, so back to jessie temporarily: https://github.com/kubernetes/kubernetes/issues/56763
     - name: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
       providerID: aws
-      kubernetesVersion: ">=1.10.0"
+      kubernetesVersion: ">=1.10.0 <1.11.0"
+    - name: kope.io/k8s-1.11-debian-jessie-amd64-hvm-ebs-2018-08-17
```
Comment (geojaz) [inline channels/stable]: i think we can resume testing against stretch as per https://github.com/kubernetes/kubernetes/pull/57896

**Comment by geojaz** [comment]
Need to fix the string checking on cos image in create_cluster_integration_test

File: channels/stable (hunk 1 above)
Comment (justinsb) [reply channels/stable]: Yes, but we should go through the alpha channel first.  Sent #5897

**Comment by justinsb** [comment]
We've done this elsewhere (rebase didn't leave any changes)

**Pull request closed without merging**